### PR TITLE
Add Slack and Matrix for contributors, too

### PR DESCRIPTION
**Pull request by beorn7**
For symmetry reasons. (o:

### CODE REVIEW

**Review by grobie — APPROVED** [review]

**Review by roidelapluie — APPROVED** [review]

File: content/community.html [hunk 1]
```diff
@@ -93,6 +93,14 @@ <h1>Contributing</h1>
       <strong>IRC:</strong>
       <code>#prometheus-dev</code> on <a href="https://libera.chat/">irc.libera.chat</a>
     </p>
+    <p>
+      <strong>Matrix:</strong>
+      <a href="https://app.element.io/#/room/#prometheus-dev:matrix.org"><code>#prometheus-dev:matrix.org</code></a>. Note: we are working on bridging matrix and libera.chat together.
+    </p>
+    <p>
+      <strong>Community-maintained slack channel:</strong>
```
Comment (RichiH) [inline content/community.html:101]:
```suggestion
      <strong>Community-maintained Slack channel:</strong>
```

File: content/community.html [hunk 2]
```diff
@@ -93,6 +93,14 @@ <h1>Contributing</h1>
...
+      <code>#prometheus-dev</code> on CNCF <a href="https://slack.cncf.io/">slack</a>.
```
Comment (RichiH) [inline content/community.html:102]:
```suggestion
      <code>#prometheus-dev</code> on CNCF <a href="https://slack.cncf.io/">Slack</a>.
```

**Review by RichiH — APPROVED** [review]
LGTM with typo fix

**Comment by beorn7** [comment]
I agree with the capitalization fix. However, I was just following the example in the Community section. 

I fixed the capitalization there, too, in #1960 . Please have a look.

**Pull request merged**